scripts/create-cluster.sh

```diff
@@ -247,6 +247,44 @@ SSH_KEY=$(cat ~/.ssh/id_installer_rsa.pub)
 #
 PULL_SECRET=$(cat ~/.pullSecret)
 
+function validate_pull_secret() {
+  set +eux
+  runtime=""
+  which docker 1>/dev/null 2>&1
+  if [[ $? == 0 ]]; then
+     runtime="docker"
+  fi
+  which podman 1>/dev/null 2>&1
+  if [[ $? == 0 && -z ${runtime} ]]; then
+     runtime="podman"
+  fi
+  if [[ -z ${runtime} ]]; then
+     echo "Warning: Neither docker nor podman is installed hence we
+     cannot check whether the pull secret is latest"
```

Comment:
You can exit here and don't need the following if test.

Reply:
Makes sense, updated the code.